spec/loadable_spec.rb

```diff
@@ -150,7 +160,8 @@
       expect(inheriting_loadable.new).not_to be_loaded
     end
 
-    it 'returns false if any load validation fails at any point in the inheritance chain' do
+    it "returns false if any load validation fails at any point \
```

Comment:
just change this entire message to `it 'returns false is any load validation fails'` Don't need the rest of the message. Plus single quotes.